internal/provider/resource_cloudflare_access_organization_test.go

```diff
@@ -0,0 +1,126 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccCloudflareAccessOrganization(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_organization.%s", rnd)
+
+	updatedName := fmt.Sprintf("%s-updated", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheck(t)
+			testAccessAccPreCheck(t)
+		},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckCloudflareAccessOrganizationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareAccessOrganizationConfigBasic(rnd, accountID),
```

Comment:
this test won't work as it will attempt to create the resource only to fail. you'll want something closer to https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/import_cloudflare_load_balancer_monitor_test.go to test that the import works and the state is saved as expected.